tests/conftest.py

```diff
@@ -167,6 +167,18 @@ def reload_modules(request: pytest.FixtureRequest):
             importlib.reload(mod)
 
 
+@pytest.fixture(scope="function")
+def chdir_tmpdir(request: pytest.FixtureRequest, tmp_path):
+    """
+    Executes a test in the tmp_path directory
+    """
+    print(tmp_path)
+    os.chdir(tmp_path)
+    print(1)
```

Comment:
Dont need these prints

Reply:
Well now :) that is embarrassing now isn't it :) 
good catch